### PR TITLE
Add crosshairs

### PR DESCRIPTION
**Pull request by AwesomestCode**
Note: this PR requires that there is an ``icons.png`` file in ``public/textures`` perfectly matching [the one in minecraft-assets](https://github.com/PrismarineJS/minecraft-assets/blob/master/data/1.16.4/gui/icons.png) to show the crosshair, otherwise it just doesn't show anything.

### CODE REVIEW

**Comment by Karang** [comment]
You can commit the texture for now, we'll remove it when including mcAssets

**Comment by AwesomestCode** [comment]
Rom told me not to, so I have to use webpack to copy it in

File: index.html [hunk 1]
```diff
@@ -67,10 +66,21 @@
...
+        height: calc(256px * 4);
+        width: calc(256px * 4);
+        transform: translate(calc(-50% + 120px * 4), calc(-50% + 120px * 4));
+        clip-path: inset(0px calc(240px * 4) calc(240px * 4) 0px);
+      }
     </style>
   </head>
-  <body>
-       <div class="chat-wrapper chat-display-wrapper">
+  <body onclick="this.requestFullscreen()">
```
Comment (rom1504) [inline index.html:81]: Why full screen ?

Comment (AwesomestCode) [reply index.html:81]: That was for testing, I can remove it if you want, but I found that it looks significantly better in full screen.

**Review by rom1504 — APPROVED** [review]

**Pull request merged**